docs/devices/TV02-Zigbee.md

```diff
@@ -39,7 +39,7 @@ pageClass: device-page
 <!-- Notes END: Do not edit below this line -->
 
 ## OTA updates
-This device supports OTA updates, for more information see [OTA updates](../guide/usage/ota_updates.md).
+This device supports OTA updates, but only using vendor Zigbee gateway (see issue [#104](https://github.com/Koenkk/zigbee-OTA/issues/104)), for more information see [OTA updates](../guide/usage/ota_updates.md).
```

Comment:
@vladyspavlov can you confirm this does not work?

Reply:
On my attempt to OTA update, I ran into the following issue: https://github.com/Koenkk/zigbee2mqtt/issues/14986

Reply:
Probably the OTA is not standard somehow, I think it would be better to remove ota support until they release an OTA which complies with the standard format.